optaplanner-quickstarts/quarkus-school-timetabling/pom.xml

```diff
@@ -202,5 +203,26 @@
         <quarkus.package.type>native</quarkus.package.type>
       </properties>
     </profile>
+    <profile> <!-- This is an example. Don't deploy it into Maven. -->
+      <id>onlyDeployWhenProductized</id>
+      <activation>
+        <property>
+          <name>!productized</name>
+        </property>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <artifactId>maven-deploy-plugin</artifactId>
+            <version>${deploy-plugin.version}</version>
+            <configuration>
+              <skip>true</skip>
+            </configuration>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
```

Comment:
We don't want to litter the quickstarts with this kind of stuff. Users do look at all the code in the quickstart and want to grok it.
However, if this only affects 7.x, and in 8.x we're solving this with the optaplanner-quickstart seperate repo - which seems to be the place - then I have no problem with this temp workaround.